src/HealthChecks.Nats/README.md

```diff
@@ -0,0 +1,49 @@
+# NATS Health Check
+
+This health check verifies the ability to communicate with a [NATS server](https://nats.io/about/). \
+It relies on `NATS.Client` package version `0.14.3` at the time this was developed. \
+Latest tag for the [official dockerhub image](https://hub.docker.com/_/nats/) is `2.6.6`.
+
+## Builder Extension
+
+```cs
+public void ConfigureServices(IServiceCollection services)
+{
+    services
+        .AddHealthChecks()
+        .AddNats(options =>
+        {
+            options.Url = "A string containing the URL (or URLs) to the NATS Server.";
```

Comment:
It would be better to provide a concrete example values (and move readable descriptions into comments).

Reply:
Commit: 403cd2af817975ded7ebe3415066f979af859ebb [403cd2a]
Parents: 7258b08ed0
nullable enable, readme